tools/mtf

```diff
@@ -34,3 +35,5 @@ while [[ $# -gt 0 ]]; do
 done
 
 avocado run $AVOCADO_ARGS
+$MTF_TOOLS/avocado_log_json.py $JSON_LOG
```

Comment:
Move to `site-packages` directory or create a script called `mtf-log-json` or similar one.

Reply:
done